test/library/standard/Random/ferguson/pcg-rand-check.chpl

```diff
@@ -32,7 +32,7 @@ writeln("Checking 32-bit RNG seq 1");
     assert( got == e32 );
   }
 
-  for (i, e32) in zip(1..6, expect32_1) {
+  for (i, e32) in zip(0..5, expect32_1) {
```

Comment:
very trivial, but this was the motivating example for me to ask whether randoms should start from 0. And I kinda hoped that this would use `zip(expect32_1.domain, expect32_1)` But it is totally fine to leave as-is

Reply:
I don't disagree, and would do similarly if I was writing you the most beautiful/maintainable program ever.  But given that even in the 1-based indexing world this test chose to use literal ranges rather than domains (for whatever reason) made me reluctant to take a different approach.